textile/features.textile

```diff
@@ -526,16 +527,16 @@ h3(#realtime-connection). Connection
 ** @(RTN20c)@ When @CONNECTING@, if the operating system indicates that the underlying internet connection is now available, the client should restart the pending connection attempt
 * @(RTN16)@ @Connection@ recovery:
```

Comment:
We also need to add a case for recovery failure ->
Currently, we cover following cases 
1. setting messageserial to zero as a part of `RTN15c7`. 
2. We never set `connectionKey` directly, it will be updated automatically for new connection.

We don't cover 
```
reverting a change made as a part of `RTN16j` -> initialized channels with their serials 
```

Reply:
it doesn't need to be reverted. the channelSerials are fine to keep. (similarly we don't clear any channel state in the case of an RTN15c7 resume failure)